src/extensions/extension-loader/extension/extension.injectable.ts

```diff
@@ -50,7 +71,7 @@ const extensionInjectable = getInjectable({
   },
 
   lifecycle: lifecycleEnum.keyedSingleton({
-    getInstanceKey: (di, instance: LensExtension) => instance,
+    getInstanceKey: (_di, instance: LensExtension) => instance,
```

Comment:
nit: please revert. We haven't been using this style in our codebase

Reply:
Sure. Should make a separate PR for this. (Avoids IDE showing a "warning": `'di' is declared but its value is never read.ts(6133)`).